### PR TITLE
Redesign feeds tab

### PR DESCRIPTION
**Pull request by ansh**
- Change the feeds icon to a hashtag
- Turn `/feeds` into...
  - A listing of the users saved feeds, followed by
  - A feed discovery listing
- Drop the multifeed
- Drop the `/search/feeds` feed discovery page

Desktop:

https://github.com/bluesky-social/social-app/assets/1270099/5c3f64c6-fca9-4823-9139-e267129d9982

Mobile:

https://github.com/bluesky-social/social-app/assets/1270099/e7068e44-20bd-4185-8374-7e314c8ee372

### CODE REVIEW

**Review by ansh — COMMENTED** [review]
Love it! I can't approve because I created the PR 🤦🏽 

But this comment is my approval

**Review by estrattonbailey — APPROVED** [review]
Fire

File: src/view/com/util/forms/SearchInput.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          onPress={onPressCancelSearchInner}
+          accessibilityRole="button"
+          accessibilityLabel="Clear search query"
+          accessibilityHint="">
+          <FontAwesomeIcon
+            icon="xmark"
+            size={16}
+            style={pal.textLight as FontAwesomeIconStyle}
+          />
+        </TouchableOpacity>
```
[the remark this answers is not in the document]
Comment (pfrazee) [reply src/view/com/util/forms/SearchInput.tsx:68]: firefox. of course.

[the remark this answers is not in the document]
Comment (estrattonbailey) [reply src/view/com/util/forms/SearchInput.tsx:68]: there are dozens of us

**Pull request merged**